### PR TITLE
fix(VStepper): fix transition in RTL

### PR DESCRIPTION
**Pull request by melbahja**
## Description
Reverse `isReverse` on rtl

## Motivation and Context
#8978 

## How Has This Been Tested?
visually

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VStepper/__tests__/VStepperContent.spec.ts [hunk 1]
```diff
@@ -57,6 +62,11 @@ describe('VStepperContent.ts', () => {
 
   it('should use reverse transition', () => {
     const wrapper = mountFunction({
+      mocks: {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VStepper/__tests__/VStepperContent.spec.ts]: please move this to `mountFunction`

Comment (sh7dm) [reply packages/vuetify/src/components/VStepper/__tests__/VStepperContent.spec.ts]: move all these to `mountFunction`

Comment (melbahja) [reply packages/vuetify/src/components/VStepper/__tests__/VStepperContent.spec.ts]: check now

File: packages/vuetify/src/components/VStepper/__tests__/VStepperContent.spec.ts [hunk 2]
```diff
@@ -72,6 +77,28 @@ describe('VStepperContent.ts', () => {
     expect(wrapper.vm.computedTransition).toBe(VTabReverseTransition)
   })
 
+  it('should use opposite of reverse transition in rtl', () => {
+    const wrapper = mountFunction({
+      mocks: {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VStepper/__tests__/VStepperContent.spec.ts]: you forgot to remove it here

Comment (melbahja) [reply packages/vuetify/src/components/VStepper/__tests__/VStepperContent.spec.ts]: this to override and test it with RTL, should `$vuetify.rtl` be `true`

Comment (sh7dm) [reply packages/vuetify/src/components/VStepper/__tests__/VStepperContent.spec.ts]: ahh, sorry :)

**Comment by sh7dm** [comment]
Thank you for your contribution and interest in improving Vuetify! Make sure to join us in the [Discord community](https://community.vuetifyjs.com).

**Pull request merged**